### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by NikitaLens**
Hometask 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # ReactJS-MentoringProgram-Q2
 ReactJS Global Mentoring Program Q2
+
+## Task 2
+
+Working branch task-2.
+
+To start, run the following commands in the console:
+```
+1. npm install
```
Comment (vladislavkovaliov) [inline README.md]:
```
npm install

npm run dev	// for dev build
npm run build	// for prod build
npm test		// for start test
npm start        // for start wepback-dev-server
```

Note: It difficult to copy only command.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang = "en">
```
Comment (vladislavkovaliov) [inline index.html]: `<html lang="en">`
`<meta charset="UTF-8">`

Note: If your page will parse by utils this spaces provide difficult for them.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+	<meta charset = "UTF-8">
```
Comment (vladislavkovaliov) [inline index.html]: `<meta charset="UTF-8">`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "reactjs-mentoring-program-q2",
+  "version": "1.0.0",
+  "description": "ReactJS Global Mentoring Program Q2",
+  "main": "webpack.config.js",
```
Comment (vladislavkovaliov) [inline package.json]: Are you sure that there is should be config.

Take a look [main](https://docs.npmjs.com/files/package.json#main), please

**Comment by vladislavkovaliov** [comment]
The purpose of using dev/prod env is split webpack's config into files base/dev/prod.
When we gonna start application in this step is going on choosing what config should we use.

Take a look [webpack](https://simonsmith.io/organising-webpack-config-environments/), please

File: package.json (hunk 4 above)
Comment (NikitaLens) [reply package.json]: Done

File: index.html (hunk 3 above)
Comment (NikitaLens) [reply index.html]: Done

File: README.md (hunk 1 above)
Comment (NikitaLens) [reply README.md]: Done

**Comment by NikitaLens** [comment]
All fix are done

**Pull request merged**